packages/office-ui-fabric-react/src/components/List/List.Props.ts

```diff
@@ -75,14 +95,48 @@ export interface IListProps extends React.HTMLAttributes<List | HTMLDivElement>
 
   /** Number of items to render. Defaults to items.length. */
   renderCount?: number;
+
+  /**
+   * Called to add styles to a given cell during rendering.
+   */
+  getCellStyle?: (item: any, index: number) => any;
```

Comment:
this returns any... what is the intent? style object? Maybe this could be something like `getCellProps` which returns `React.HTMLAttributes` or something.